backend/src/admin/admin.service.ts

```diff
@@ -16,6 +18,10 @@ export class AdminService {
 
     register = async (registrationData: RegisterAdminDto) => this.authenticationService.registerAdmin(registrationData);
     login = async (loginData: LoginAdminDto) => this.authenticationService.loginAdmin(loginData);
+    registerJunior = async (registrationData: RegisterJuniorDto) => this.authenticationService.registerJunior(registrationData);
```

Comment:
Im also unsure how I feel about this. Is this actually needed and the "nest way"?  Cant we inject authService to admin-controller and use it directly there?

Reply:
We could, I was here considering making the controllers as "dumb" as possible, so each controller relies only on its associated service.